src/legacy/core_plugins/console/public/src/controllers/sense_controller.js

```diff
@@ -94,7 +114,9 @@ module.controller('SenseController', function SenseController(Private, $scope, $
 
   $scope.sendSelected = () => {
     input.focus();
-    input.sendCurrentRequestToES();
+    input.sendCurrentRequestToES(() => {
+      $scope.historyDirty = new Date().getTime();
```

Comment:
Could you walk me through the role `$scope.historyDirty` plays in the UX?

Reply:
It's used to force the `sense-history` to regenerate in case a user runs a query, while the history panel is open.

Reply:
Ah, I see, thanks! Could we please rename it to `lastRequestTimestamp`, and add a comment on this line:

```js
// History watches this value and will re-render itself when it changes, so that
// the list of requests stays up-to-date as new requests are sent.
```